internal/query/relations.go

```diff
@@ -760,7 +760,8 @@ func (e *engine) relationshipsToNonRoles(rels []*pb.Relationship) ([]types.Relat
 	var out []types.Relationship
 
 	for _, rel := range rels {
-		if rel.Subject.Object.ObjectType == e.namespace+"/role" {
+		// skip relationships for v1 roles, and wildcard relationships for v2 roles
+		if rel.Subject.Object.ObjectType == e.namespace+"/role" || rel.Subject.Object.ObjectId == "*" {
```

Comment:
Magic strings here feel like they're going to introduce problems. We have a configurable role resource but then have `/role` hard-coded here.

Reply:
only v2 roles are configurable, v1 role will remain hard coded as `role`

Reply:
`git blame` 😈